main.py

```diff
@@ -0,0 +1,70 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def create_author(
+        author: schemas.AuthorCreate,
+        db: Session = Depends(get_db)
+):
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def get_authors(
+        skip: int = 0,
+        limit: int = 10,
+        db: Session = Depends(get_db),
+):
+    return crud.get_authors(db=db, skip=skip, limit=limit)
+
+
+@app.get("/authors/{id}/", response_model=schemas.Author)
+def get_author_by_id(
+        id: int,
+        db: Session = Depends(get_db),
+):
+    author = crud.get_author_by_id(db=db, id=id)
```

Comment:
The parameter `id` in the `get_author_by_id` function is being passed to `crud.get_author_by_id` as `id=id`. However, the `crud.get_author_by_id` function expects `author_id` as the parameter name. You should change `id=id` to `author_id=id` to match the expected parameter name in the CRUD function.